src/components/SingleClubEvent.jsx

```diff
@@ -1,37 +1,38 @@
 import * as React from "react";
 import ClubUpperImage from "../assets/pictures/ClubUpperImage.svg";
-import { useLocation, useNavigate } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 import "react-toastify/dist/ReactToastify.css";
 import "../styles/ClubsPage.css";
-import displayRazorpay from "../service/PaymentGateway";
+// import displayRazorpay from "../service/PaymentGateway";
+import Button from "./Button";
 
 export default function SingleClubEvent({ club, type, event }) {
-  const [money, setmoney] = React.useState({ donatedmoney: 0 });
+  // const [money, setmoney] = React.useState({ donatedmoney: 0 });
 
-  const location = useLocation();
+  // const location = useLocation();
   const nav = useNavigate();
 
   const cardButtonHandler = () => {
     nav(`/clubs/${club._id}`);
   };
-  const expand = () => {
-    document.getElementById(`less${club._id}`).classList.add("hidden");
-    document.getElementById(`more${club._id}`).classList.remove("hidden");
-  };
+  // const expand = () => {
```

Comment:
You can remove these commented codes as they were previously been used but not now at all.